src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1852,7 +1852,7 @@ private bool IsInTypeofExpression(int position)
         private static void AddUnwrappingErrorTypes(ArrayBuilder<Symbol> builder, Symbol s)
         {
             var originalErrorSymbol = s.OriginalDefinition as ErrorTypeSymbol;
-            if ((object)originalErrorSymbol != null)
+            if ((object)originalErrorSymbol != null && originalErrorSymbol.CandidateSymbols != null)
```

Comment:
@cston are you making a comment about what this PR should do or that possibly `ErrorTypeSymbol` is the source of the bug? Can't tell from the comment.